controllers/subscription.go

```diff
@@ -55,10 +55,11 @@ func newSubscription(p api.Pattern) *operatorv1alpha1.Subscription {
 	//          value: "*"
 
 	spec := &operatorv1alpha1.SubscriptionSpec{
-		CatalogSource:          p.Spec.GitOpsConfig.OperatorSource,
-		CatalogSourceNamespace: "openshift-marketplace",
-		Channel:                p.Spec.GitOpsConfig.OperatorChannel,
-		Package:                "openshift-gitops-operator",
+		CatalogSource:          GitOpsDefaultCatalogSource,
+		CatalogSourceNamespace: GitOpsDefaultCatalogSourceNamespace,
+		Channel:                GitOpsDefaultChannel,
+		Package:                GitOpsDefaultPackageName,
+		StartingCSV:            "",
```

Comment:
Didn't we talk about not doing this and just creating it properly the first time?

Reply:
Yup. Re-implemented in latest updates.